quinn/src/connection.rs

```diff
@@ -741,6 +743,11 @@ where
                         let _ = finishing.send(stop_reason.map(WriteError::Stopped));
                     }
                 }
+                Stream(StreamEvent::Stopped { id, stop_reason }) => {
```

Comment:
A stopped stream and a reset stream are different things. A stream is stopped when the receiver tells the sender it's no longer interested (via `STOP_SENDING`). A stream is reset when the sender tells the receiver it's aborted sending on that stream (via `RESET_STREAM`).

Reply:
This is why this variant is called Stopped. I don't see your point here?

Reply:
The term `reset` is used in the body of this case.

Reply:
Oh! yeah that is an artifact! -_- 
Thanks!